src/styles/components/TextEditor.styl

```diff
@@ -0,0 +1,121 @@
+.TextEditor
```

Comment:
@hudakdidit I did a bunch of small css tweaks that could've already fixed that, I don't see any extra space underneath
<img width="971" alt="screen shot 2017-05-28 at 2 35 11 pm" src="https://cloud.githubusercontent.com/assets/1121873/26532343/e8d20bb4-43b2-11e7-9bee-097176ddfff6.png">